pages/docs/manual/latest/primitive-types.mdx

```diff
@@ -168,7 +168,7 @@ As with integers, you may use underscores within literals to improve readability
 
 ## Unit
 
-The `unit` type is a type that indicates the absense of a specific value. It has only a single value, `()`, which acts as a placeholder when no other value exists or is needed. It compiles to JavaScript's `undefined` and resembles the `void` type in languages such as C++. What's the point of such a type?
+The `unit` type indicates the absense of a specific value. It has only a single value, `()`, which acts as a placeholder when no other value exists or is needed. It compiles to JavaScript's `undefined` and resembles the `void` type in languages such as C++. What's the point of such a type?
```

Comment:
And I thought we were having communication problems. Or did the AI find that mistake?

Reply:
Since you summoned the AI:

>Hi @jmagaram,
>
>Thanks for your contribution to the project! It's great to see your enthusiasm and effort. Regarding the typo, it's easy to overlook small things like that. No worries - we all learn from our mistakes.
>
>As you continue contributing, remember that we're all here to support each other. Feel free to reach out if you have any questions or need guidance. Your input is valuable, and we appreciate your work on this project. To make our collaboration even more efficient, consider being concise in your descriptions and take a moment to review your posts before submitting. This will help keep our discussions focused and productive.